# About me
## [Doctoral thesis - Cells in Time and Space: Beyond comprehensive cellular atlases towards a deeper understanding of disease](https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1929647&dswid=-2336)
My thesis centered around spatial biology by developing and applying in situ sequencing (ISS) to map cellular organization and disease processes directly within intact tissues. [Paper I](https://pmc.ncbi.nlm.nih.gov/articles/PMC7641728/) introduces an improved hybridization-based ISS method with higher sensitivity and broader gene coverage, enabling analysis of challenging human brain tissue. [Paper II](https://www.nature.com/articles/s42003-021-02517-z) uses this technology to chart 75 transcriptomic cell types across the human cortex, mapping nearly 60,000 cells in their native laminar context. [Paper III](https://pmc.ncbi.nlm.nih.gov/articles/PMC9131697/) applies ISS to developmental biology, profiling oligodendrocyte lineage progression across mouse spinal cord and brain, revealing spatially heterogeneous maturation and neighbor relationships. [Paper IV](https://doi.org/10.1016/j.cell.2024.02.030) models neuroinflammatory lesion development in EAE, constructing a spatiotemporal atlas of MS-like pathology and uncovering globally induced, reversible disease-associated glial states and lesion-propagation dynamics, validated in human MS spinal cord. Collectively, the thesis demonstrates how ISS can move beyond static cellular atlases to illuminate dynamic tissue architecture, lineage progression, and disease evolution.

## Main PhD project
- **[Cellular Architecture of Evolving neuroinflammatory lesions and MS pathology](https://github.com/christoffermattssonlangseth/cellular-architecture-of-evolving-neuroinflammatory-lesions-and-multiple-sclerosis-pathology)**: Multiple sclerosis (MS) involves multifocal lesions and chronic smoldering pathology, yet the dynamic cellular processes driving lesion evolution remain poorly understood. Using the EAE mouse model together with single-cell spatial transcriptomics (in situ sequencing), we mapped disease progression across time and space, revealing centrifugal growth of active lesions and the emergence of disease-associated glial states that form independently of lesions and change dynamically throughout the disease course. Parallel single-cell spatial mapping of human MS spinal cords confirmed distinct distributions of homeostatic and disease-associated glia, enabled fine-grained deconvolution of active and inactive lesions, and uncovered previously unrecognized pathological sub-compartments. Together, this work establishes a cross-species single-cell spatial atlas of MS neuropathology and highlights the complex, evolving cellular dynamics underlying lesion initiation, propagation, and resolution. (**first ever Xenium publication**) *([Cell, 2024](https://doi.org/10.1016/j.cell.2024.02.030))* <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

Featured in:
1. [10X Genomics website on inital release](https://www.10xgenomics.com/blog/plotting-the-course-of-multiple-sclerosis-lesions-with-single-cell-spatial-imaging)
2. [Stockholm University website](https://www.su.se/department-of-biochemistry-and-biophysics/news/cellular-architecture-of-lesions-in-ms-now-mapped-out-1.723838)
3. [Karolinska Institute website](https://news.ki.se/cellular-architecture-of-lesions-in-ms-now-mapped-out)

## PostDoc projects
- **[oligo-mtDSB](https://github.com/christoffermattssonlangseth/oligo-mtDSB)**: Investigates the consequences of DBS in mtDNA in oligos. Xenium data. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[metamitoMicS](https://github.com/christoffermattssonlangseth/metamitoMicS)**: metamitoMicS is a repository focused on the analysis of metabolic and mitochondrial pathways in neuroinflammation and MS. It brings together spatial transcriptomics, omics integration, and computational biology to explore how energetic dysfunction shapes lesion microenvironments. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[RRmap (Private repo for now)](https://github.com/christoffermattssonlangseth/RRMap)**: RRMap: Relapsing-Remitting Mapping of CNS Inflammation. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[OligoC4b](https://github.com/christoffermattssonlangseth/OligoC4b)**: Exploring C4b in the context of disease and aging. Contains open source data from Xenium, Visum and single-cell RNA seq. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[ST-BRICHOS](https://github.com/christoffermattssonlangseth/ST-BRICHOS)**: Spatial transcriptomics of AD mouse brains treated with BRICHOS to map amyloid, neuroinflammatory, and neuroprotective transcriptional changes. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[bejsor](https://github.com/christoffermattssonlangseth/bejsor)**: This is a modified fork, humorously named Bejsor, which includes a stupid workaround to reduce excessive RAM usage when running Baysor on high-transcript-count datasets
 <img src="https://www.logo.wine/a/logo/Julia_(programming_language)/Julia_(programming_language)-Logo.wine.svg" height="20" class="image is-16x16 is-inline-block" />


