# About me
Computational biologist in the Castelo-Branco lab at Karolinska Institutet, specializing in neuroinflammation and spatial omics. Iâ€™m particularly interested in how spatially resolved technologies can uncover metabolic and mitochondrial dynamics within inflamed CNS tissue.

## ðŸ“„ CV & Research Vision
- **Full CV**: [CV](https://christoffermattssonlangseth.github.io/cv.html)
- **Research vision repo**: [research-vision](https://github.com/christoffermattssonlangseth/research-vision)


## [Doctoral thesis â€“ Cells in Time and Space: Beyond comprehensive cellular atlases towards a deeper understanding of disease](https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1929647&dswid=-2336)

My doctoral thesis focused on developing and applying **in situ sequencing (ISS)** to study cellular organization, lineage progression, and disease processes directly within intact tissue.  
[Paper I](https://pmc.ncbi.nlm.nih.gov/articles/PMC7641728/) presents an improved hybridization-based ISS method with increased sensitivity and gene coverage, enabling robust analysis of human brain tissue.  
[Paper II](https://www.nature.com/articles/s42003-021-02517-z) maps transcriptomic cell types across the human cortex, resolving cellular identities within their native laminar architecture.  
[Paper III](https://pmc.ncbi.nlm.nih.gov/articles/PMC9131697/) applies ISS to developmental biology, charting oligodendrocyte lineage progression across mouse brain and spinal cord and revealing spatially heterogeneous maturation.  
[Paper IV](https://doi.org/10.1016/j.cell.2024.02.030) constructs a spatiotemporal atlas of neuroinflammatory lesion development in EAE, identifying reversible disease-associated glial states and lesion dynamics, with validation in human multiple sclerosis tissue.  

Collectively, the thesis demonstrates how spatial transcriptomics can move beyond static cellular atlases to reveal dynamic tissue organization and disease evolution in situ.

## Main PhD project
- **[Cellular Architecture of Evolving Neuroinflammatory Lesions and MS Pathology](https://github.com/christoffermattssonlangseth/cellular-architecture-of-evolving-neuroinflammatory-lesions-and-multiple-sclerosis-pathology)**:  
Using single-cell spatial transcriptomics in the EAE model and human MS spinal cord, this work maps the spatiotemporal evolution of neuroinflammatory lesions, identifies dynamically regulated disease-associated glial states, and reveals previously unrecognized pathological sub-compartments, establishing a cross-species single-cell spatial atlas of MS pathology.  
(**first ever Xenium publication**) *([Cell, 2024](https://doi.org/10.1016/j.cell.2024.02.030))* <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

Featured in:
1. [10X Genomics website on inital release](https://www.10xgenomics.com/blog/plotting-the-course-of-multiple-sclerosis-lesions-with-single-cell-spatial-imaging)
2. [Stockholm University website](https://www.su.se/department-of-biochemistry-and-biophysics/news/cellular-architecture-of-lesions-in-ms-now-mapped-out-1.723838)
3. [Karolinska Institute website](https://news.ki.se/cellular-architecture-of-lesions-in-ms-now-mapped-out)

## PostDoc projects
- **[RRmap (Private repo for now)](https://github.com/christoffermattssonlangseth/RRMap)**: RRMap: Relapsing-Remitting Mapping of CNS Inflammation. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[oligo-mtDSB](https://github.com/christoffermattssonlangseth/oligo-mtDSB)**: Investigates the consequences of DBS in mtDNA in oligos. Xenium data. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[metamitoMicS](https://github.com/christoffermattssonlangseth/metamitoMicS)**: metamitoMicS is a repository focused on the analysis of metabolic and mitochondrial pathways in neuroinflammation and MS. It brings together spatial transcriptomics, omics integration, and computational biology to explore how energetic dysfunction shapes lesion microenvironments. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[OligoC4b](https://github.com/christoffermattssonlangseth/OligoC4b)**: Exploring C4b in the context of disease and aging. Contains open source data from Xenium, Visum and single-cell RNA seq. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[ST-BRICHOS](https://github.com/christoffermattssonlangseth/ST-BRICHOS)**: Spatial transcriptomics of AD mouse brains treated with BRICHOS to map amyloid, neuroinflammatory, and neuroprotective transcriptional changes. <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[Spinal cord injury](https://github.com/christoffermattssonlangseth/spinal_cord_injury)**: MERSCOPE data targetting 500 genes across spinal cord injury models <img src="https://raw.githubusercontent.com/abrahamcalf/programming-languages-logos/master/src/python/python.svg" height="16" class="image is-16x16 is-inline-block" />

- **[bejsor](https://github.com/christoffermattssonlangseth/bejsor)**: This is a modified fork, humorously named Bejsor, which includes a stupid workaround to reduce excessive RAM usage when running Baysor on high-transcript-count datasets. <img src="https://www.logo.wine/a/logo/Julia_(programming_language)/Julia_(programming_language)-Logo.wine.svg" height="20" class="image is-16x16 is-inline-block" />

